Processes/Docs-Contribution-Process-Internal.md

```diff
@@ -0,0 +1,105 @@
+
+
+# Docs Contribution Process - Internal
+
+## Overview
+This document provides instructions for contributors internal to the ENF Foundation on how to contribute new or updated content to EOS Documentation (https://docs.eosnetwork.com).
+
+ You can rely on the DevRel team to perform a thorough edit and review of your content before it is published to the EOS Documentation portal. The publication process includes GitHub workflows to publish Markdown (.md) formatted content.
+
+As a content contributor you are responsible for the accuracy and completeness of your submitted documentation. The DevRel team may contact you for clarification of your contribution, if necessary. You are also responsible to identify future changes for your content and let the DevRel team know of any information maintenance issues.
+
+As a content contributor you are also responsible to review the final submission before it is published. Your review is reflective of the accuracy and completeness of the content.
+
+### When to Use this Procedure
```

Comment:
Both of these topics are standard technical writing best practices. I think they are most useful when part of a procedure or process because they help frame the content.  

If the reader or doer of the procedure gets a result other than the expected result, they may need to consider  their input to the procedure or some of their decisions to determine if the procedure worked as intended for them.

Reply:
Done